doc/cmp.txt

```diff
@@ -380,6 +415,62 @@ completion.autocomplete~
   nvim-cmp does not completion automatically but you can still use the manual
   completion though.
 
+                                           *cmp-config.window.completion.border*
+window.completion.border~
+  `string | string[] | nil`
+  Border characters used for the completion popup menu when
+  |experimental.native_menu| is disabled.
+
+                                        *cmp-config.window.completion.scrollbar*
+window.completion.scrollbar~
+  `string | false`
+  A setting defining which character to use for the scrollbar.
+
+  * If an empty string (`""`), |hl-PmenuThumb| will be used.
+  * If `false`, no scrollbar will be rendered.
+  * If |experimental.native_menu| is `true`, this setting will be ignored.
+
+  Default: `""`
+  Recommended: `"║"`
+
+                                               *cmp-config.window.documentation*
+window.documentation~
+  `false | cmp.DocumentationConfig`.
+  If set to `false`, the documentation of each item will not be shown.
+  Else, a table representing documentation configuration should be provided.
+  The following are possible options.
+
+                                        *cmp-config.window.documentation.border*
+window.documentation.border~
+  `string | string[] | nil`
+  Border characters used for documentation window.
+
+                                      *cmp-config.window.documentation.maxwidth*
+window.documentation.maxwidth~
+  `number`
+  The documentation window's max width.
+
+                                     *cmp-config.window.documentation.maxheight*
+window.documentation.maxheight~
+  `number`
+  The documentation window's max height.
+
+                                     *cmp-config.window.documentation.scrollbar*
+window.documentation.scrollbar~
+  `string`
+  A setting defining which character to use for the scrollbar.
+
+  * If an empty string (`""`), |hl-PmenuThumb| will be used.
+  * If `false`, no scrollbar will be rendered.
+
+  Default: `""`
+  Recommended: `"║"`
+
+                                        *cmp-config.window.documentation.zindex*
+window.documentation.zindex~
+  `number`
+  The documentation window's zindex.
```

Comment:
I like consistency. Therefore, I would like to add z-index to the `window.completion` section too.

But max_width/max_height is... hm... It's hard to think proper way.